delocate/tests/test_wheeltools.py

```diff
@@ -118,70 +134,62 @@ def assert_winfo_similar(whl_fname, exp_items, drop_version=True):
     assert_equal(len(exp_items), len(w_info))
     # Extract some information from actual values
     wheel_parts['pip_version'] = dict(w_info)['Generator'].split()[1]
-    for (key1, value1), (key2, value2) in zip(exp_items, w_info):
+    for (key1, value1), (key2, value2) in zip(sorted(exp_items),
+                                              sorted(w_info)):
```

Comment:
I've noticed a lot of effort is needed to keep `exp_items` and `w_info` in sync.  Do you think these would've been better as dictionaries?

Reply:
The difficulty is the multiple values for tags.  It would make it a little easier to have a dictionary with key `tags` and values of tag tuples, and do the manipulation on that.  Maybe that's what you meant.

Reply:
I didn't consider that there could be duplicate keys.

`get_info` seems to return `email.message.EmailMessage` instead of a dict, which caused me to make a series of faulty assumptions.